scripts/bundleSize.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Size limit for all bundles used by each service (K)
+# Keep these +/- 5K and update frequently!
+min=684
+max=702
+
+services=( "news" "persian" "igbo" "yoruba" "pidgin" )
+failure=false
+
+for service in ${services[@]}
+do
+   size=$(($(cat build/public/static/js/{main,vendor,$service}-*.js | wc -c | tr -d ' ') / 1000 ))
+
+   if [[ $size -lt $min ]]; then
+     tput setaf 1
+     echo "Bundle size for $service is too small at ${size}K, please update thresholds"
+     failure=true
+   elif [[ $size -gt $max ]]; then
+     tput setaf 1
+     echo "Bundle size for $service is too great at ${size}K, please update thresholds"
+     failure=true
+   else 
+    tput setaf 2
+    echo "${service} JS = ${size}K"
+   fi
+done
+
+tput sgr0
+
+if [ $failure = true ] ; then
+    exit 1;
```

Comment:
For a future issue, Id like this to comment on/update a status on the PR from travis, stating the bundle Size. This will make it far easier for us to track down what PR's have negatively affected the bundle sizes. 

Nice work on this one Drew! 👍

Reply:
Yeah def :) For now, on Benjamins instruction ive also made the limits purposely very tight so we should easily spot in history who changed them :)
```
# Keep these +/- 5K and update frequently!
min=684
max=702
```